pyfr/backends/base/makoutil.py

```diff
@@ -154,8 +154,8 @@ def _parse_expand_args(name, mparams, margsig, args, kwargs):
 
     # Separate kwargs into params and Python data params
     if unknown := set(kwargs) - set(mparams) - set(margs):
-        unknown = unknown.pop()
-        raise ValueError(f'Unknown parameter "{unknown}" in macro "{name}"')
+        raise ExceptionGroup(f'In macro: {name}',
+                           [ValueError(f'Unknown param "{unknown.pop()}"')])
```

Comment:
Might as well use a list comprehension here to nab all of the unknown params and raise a value error for each one rather than just popping.